monai/utils/module.py

```diff
@@ -60,6 +60,16 @@
     "pytorch_after",
 ]
 
+WRITER_PACKAGE_MAP: Dict[str, List[str]] = {
+    "png": ["pillow"],
+    "jpg": ["pillow"],
+    "jpeg": ["pillow"],
+    "nii": ["nibabel"],
+    "nii.gz": ["nibabel"],
+    "dcm": ["pydicom"],
+    # Add more mappings as needed
```

Comment:
Who is supposed to add the rest of the entries on this list? [ITK](https://itk.org/Wiki/ITK/File_Formats) supports quite a few, including all of the ones you listed above.

Reply:
We can add ITK for a number of these types though the listed libraries are the primary ones used. If you can identify what other libraries load each of these types (and other types that can be loaded) please add them here as well.